### DISCUSSION TITLE
Some questions about torch_geometric.datasets

### DISCUSSION THREAD

**Discussion by SongZan222** [pyg-team/pytorch_geometric#7178] [Q&A]
Dear PyG community,
What data set under this class has node labels and a higher average number of nodes?
Are there other data sets like  **torch_geometric.datasets.TUDataset** ?

**Comment by EdisonLeeeee** [comment]
Please refer to [dataset cheatsheet](https://pytorch-geometric.readthedocs.io/en/latest/cheatsheet/data_cheatsheet.html) for more information.
